code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -349,3 +350,49 @@
 	righthand_file = 'icons/mob/inhands/equipment/hydroponics_righthand.dmi'
 	volume = 100
 	list_reagents = list(/datum/reagent/toxin/plantbgone = 100)
+
+/obj/item/reagent_containers/spray/syndicate
+	name = "suspicious spray bottle"
+	desc = "A spray bottle, with a high performance plastic nozzle. The color scheme makes you feel slightly uneasy."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "sprayer_sus_8"
+	item_state = "sprayer_sus"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	spray_range = 4
+	stream_range = 2
+	volume = 100
+	custom_premium_price = 200
+
+/obj/item/reagent_containers/spray/syndicate/Initialize()
+	. = ..()
+	icon_state = pick("sprayer_sus_1", "sprayer_sus_2", "sprayer_sus_3", "sprayer_sus_4", "sprayer_sus_5","sprayer_sus_6", "sprayer_sus_7", "sprayer_sus_8")
+
+/obj/item/reagent_containers/spray/medical
+	name = "medical spray bottle"
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "sprayer_med_red"
+	item_state = "sprayer_med_red"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	volume = 100
+	unique_reskin = list("Red" = "sprayer_med_red",
+						"Yellow" = "sprayer_med_yellow",
+						"Blue" = "sprayer_med_blue")
+
+/obj/item/reagent_containers/spray/medical/AltClick(mob/user)
+	if(unique_reskin && !current_skin && user.canUseTopic(src, BE_CLOSE, NO_DEXTERY))
+		reskin_obj(user)
+		switch(icon_state)
+			if("sprayer_med_red")
+				item_state = "sprayer_med_red"
+				to_chat(user, "item state set.'")
+				return
+			if("sprayer_med_yellow")
+				item_state = "sprayer_med_yellow"
+				to_chat(user, "item state set.'")
+				return
+			if("sprayer_med_blue")
+				item_state = "sprayer_med_blue"
+				to_chat(user, "item state set.'")
+				return
```

Comment:
are these debug messages?

reskin_obj should handle setting the item_state too imo.

Reply:
Yes, good catch

Reply:
I have removed the debug printout, not sure how to best tackle the reskin_obj thing. I agree that allowing reskin_obj handle item state as well makes sense but i am not sure i am the right man for the job.

 I might be able to implement it competently with some help with the good people of discord.

Reply:
i forgot what item_state does but you can just add an arg to reskin_obj prob.